Hardcore.lua

```diff
@@ -17,6 +17,11 @@ You should have received a copy of the GNU General Public License
 along with the Hardcore AddOn. If not, see <http://www.gnu.org/licenses/>.
 --]]
 
+--[[ Const variables ]]--
+local OFF = 0
+local PVP = 1
+local PVP_AND_ENEMY_FACTION = 2
```

Comment:
Am I missing something? Where can the user change these? Seems to be user configuration.

Also Nitpick: Naming is a bit off here. 
`grief_warning_conditions`,
`OFF`
`PVP`
`PVP_AND_ENEMY_FACTION`

Can I suggest a consistentcy with `grief_warning_conditions`?
`GRIEF_WARNING_OFF`
`GRIEF_WARNING_PVP`
`GRIEF_WARNING_PVP_AND_ENEMY_FACTION`.

Reply:
Suggestions sounds great, I'll change those later! I assume this is something people want to toggle so i was just going to wait for interface options to merge

Reply:
For now we can add this as a slash command if were not ready to build UI out for it